src/components/CommitBox.tsx

```diff
@@ -114,29 +127,52 @@ export class CommitBox extends React.Component<ICommitBoxProps> {
           className={commitSummaryClass}
           type="text"
           placeholder={summaryPlaceholder}
-          title={this.props.trans.__(
-            'Enter a commit message summary (a single line, preferably less than 50 characters)'
-          )}
+          title={
+            this.props.amend
+              ? this.props.trans.__(
+                  'You are amending this commit to the previous one, so the previous commit summary will be used'
```

Comment:
Let's make it shorter:

```suggestion
                  'Amending the commit will re-use the previous commit summary'
```